ClearKeep/App/DependencyResolver/DependencyResolver.swift

```diff
@@ -34,21 +34,15 @@ class DependencyResolver {
 		// MARK: - CommonUI
 		CommonUI.DependencyResolver.shared = CommonUI.DependencyResolver(CommonUIConfig(fontSet: fontSet, colorSet: colorSet, imageSet: imageSet))
 		
+		// MARK: - Chat Secure
+		channelStorage = ChannelStorage(config: ConfigurationProvider.default)
+		ChatSecure.DependencyResolver.shared = ChatSecure.DependencyResolver(channelStorage)
+		
 		// MARK: - Services
 		securedStoreService = SecuredStoreService()
 		persistentStoreService = PersistentStoreService()
 		appTokenService = AppTokenService(securedStoreService: securedStoreService, persistentStoreService: persistentStoreService)
 		
-		let networkingService = NetworkingService(configurations: NetworkConfigurations(),
-												  httpHeaderHandler: NetworkHTTPHeaderHandler(tokenService: appTokenService),
-												  responseHandler: NetworkHTTPResponseHandler(),
-												  networkConnectionHandler: NetworkConnectionHandler())
-		
-		let apiConfig = APIConfigurations()
-		let queryAdapter = APIResourceQueryAdapter(config: apiConfig, tokenService: appTokenService)
-		let responseAdapter = APIResourceResponseAdapter(jsonHandler: JSONDataHandler())
-		
-		sampleAPIService = APIService(client: networkingService, query: queryAdapter, resourceHandler: responseAdapter)
 		
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/03ed79698ba6e18faa1dddb6e6524285a668b75e/ClearKeep/App/DependencyResolver/DependencyResolver.swift#L46"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>DependencyResolver.swift:46</code></td>
    </tr>
  </tbody>
</table>